src/mjolnir/pbfgraphparser.cc

```diff
@@ -748,6 +748,14 @@ struct graph_callback : public OSMPBF::Callback {
       restriction.set_modes(kTruckAccess);
       osmdata_.access_restrictions.insert(
           AccessRestrictionsMultiMap::value_type(osmid_, restriction));
+    };
+     tag_handlers_["maxaxles"] = [this]() {
+      OSMAccessRestriction restriction;
+      restriction.set_type(AccessType::kMaxAxles);
+      restriction.set_value(std::stoul(tag_.second));
```

Comment:
@kevinkreiser Thanks. I am not a C dev, so any help would be much appreciated here. I believe your were correct with this recommendation. Additionally, I changed the graph.lua to parse the kv to a number, and the truckcost.cc to a `cast int`.  Let me know what else I might be missing here. Or if there is anything larger I really need to get this merged. Thanks again for the help man. Much appreciated.